charts/argo-templates/templates/ingest-file.yaml

```diff
@@ -96,8 +104,6 @@ spec:
             valueFrom:
               parameter: '{{ "{{tasks.get-job-result.outputs.result}}" }}'
               default: 'null'
-    {{- $secretVolume := "sa-secret-volume" }}
-    {{- $mountPath := "/secret" }}
```

Comment:
The poll-job templates needs access to these variables, and I don't think it makes sense to hard code them in a fourth place, so I added them as inputs to these templates. I'm not sure though what the best place to define them is (a variable in `clinvar-ingest`? Or a variable here?). Let me know if anyone has thoughts.

Reply:
Even though these hard-coded values were used across multiple steps, they aren't actually linked across steps (each pod generates a new "fake" volume using the same short-hand name). Reusing the name was just for convenience, but I'd be fine with inlining the values across the board.